jpo-geojsonconverter/pom.xml

```diff
@@ -90,26 +90,37 @@
         </dependency>
 
         <!-- Kafka Dependencies -->
+        <dependency>
+            <groupId>org.springframework.kafka</groupId>
+            <artifactId>spring-kafka</artifactId>
+            <version>2.8.11</version>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.kafka</groupId>
+            <artifactId>spring-kafka-test</artifactId>
+            <version>2.8.11</version>
```

Comment:
I don't think it's necessary to specify version numbers for Spring dependencies, they ought to automatically use versions compatible with the spring parent if omitted.